src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -12,54 +9,6 @@ public enum Priority {
         SECONDARY
     }
 
-    /**
-     * Fund your EasyPost wallet by charging your primary or secondary card on file.
-     *
-     * @param amount amount to fund.
-     * @param primaryOrSecondary primary or secondary payment method.
-     * @return CreditCardFund object.
-     * @throws EasyPostException when the request fails.
-     */
-    public static CreditCardFund fund(String amount, Priority primaryOrSecondary) throws EasyPostException {
```

Comment:
We still need these functions, but rather than returning a `CreditCardFund` object, they will return the boolean `true` (since any errors occurring will trigger an exception upstream)